packages/modeling/src/primitives/polyhedron.js

```diff
@@ -32,13 +35,25 @@ const polyhedron = (options) => {
   if (faces.length < 1) {
     throw new Error('one or more faces are required')
   }
+  if (colors) {
+    if (!Array.isArray(colors)) {
+      throw new Error('colors must be an array')
+    }
+    if (colors.length !== faces.length) colors = undefined
```

Comment:
I think this should throw an error, ie as a user you need to be able to know why the colors are not being displayed :)

Reply:
Done